examples/vortexrings/vortexrings_functions.jl

```diff
@@ -386,22 +386,22 @@ function calc_elliptic_radius(outRm, outRp, Z, pfield, nrings, intervals;
         for pi in (intervals[ri]+1):(intervals[ri+1])
 
             P = vpm.get_particle(pfield, pi)
-            weightx = dot(P.Gamma, unity)
-            weighty = dot(P.Gamma, unitx)
+            weightx = dot(P[4:6], unity)
```

Comment:
Indexing [4:6] will allocate; should probably use get_Gamma or a view

Reply:
Resolved in [2a3435a](https://github.com/byuflowlab/FLOWVPM.jl/pull/13/commits/2a3435ae13ee51e563ecc985dd8daa266c1e95cd)